lib/adagrams.rb

```diff
@@ -0,0 +1,183 @@
+#Wave 1
+# require 'pry'
+
+BANANA_BAG =[ {"A" => 9}, {"B" => 2}, {"C" => 2}, {"D" => 4}, {"E" => 12}, {"F" => 2}, {"G" => 3}, {"H" => 2}, {"I" => 9}, {"J" => 1}, {"K" => 1}, {"L" => 4}, {"M" => 2}, {"N" => 6}, {"O" => 8}, {"P" => 2}, {"Q" => 1}, {"R" => 6}, {"S" => 4}, {"T" => 6}, {"U" => 4}, {"V" => 2}, {"W" => 2}, {"W"  => 1}, {"Y" => 2}, {"Z" => 1} ]
+
+def making_tiles (array_of_hashes)
+  bag = []
+  letters = []
+  values = []
+  array_of_hashes.each do |hash|
+    hash.each do |key, value|
+      letters << key
+      values << value
+    end
+    # puts values
+  end
+  26.times do |i|
+    current_bag = letters[i] * values[i]
+    bag << current_bag.split("")
+  end
+
+  return bag
+
+end
+
+test = making_tiles(BANANA_BAG)
+BIG_BAG = test.flatten
```

Comment:
normally, code that isn't located within a method won't run unless another part of the program calls it.

This code ends up executing when we run `rake` because our tests load this code and execute it... in general, I would expect these two lines to be the first lines in the `draw_letters` method actually